trulens_eval/trulens_eval/streamlit.py

```diff
@@ -0,0 +1,237 @@
+import asyncio
+import json
+import math
+from typing import List
+
+import pandas as pd
+
+# https://github.com/jerryjliu/llama_index/issues/7244:
+asyncio.set_event_loop(asyncio.new_event_loop())
+
+from millify import millify
+from pydantic import BaseModel
+import streamlit as st
+from streamlit_pills import pills
+
+from trulens_eval import Tru
+from trulens_eval.database.legacy.migration import MIGRATION_UNKNOWN_STR
+from trulens_eval.react_components.record_viewer import record_viewer
+from trulens_eval.schema.app import AppDefinition
+from trulens_eval.schema.feedback import FeedbackCall
+from trulens_eval.schema.record import Record
+from trulens_eval.utils import display
+from trulens_eval.utils.json import json_str_of_obj
+from trulens_eval.utils.python import Future
+from trulens_eval.ux import styles
+from trulens_eval.ux.components import draw_metadata
+
+
+class FeedbackDisplay(BaseModel):
+    score: float = 0
+    calls: List[FeedbackCall]
+    icon: str
+
+
+def trulens_leaderboard(app_ids: List[str] = None):
+    """
+    Render the leaderboard page.
+
+    Args:
+
+        app_ids List[str]: A list of application IDs (default is None)
+
+    !!! example
+
+        ```python
+        from trulens_eval import streamlit as trulens_st
+
+        trulens_st.trulens_leaderboard()
+        ```
+    """
+    tru = Tru()
+
+    lms = tru.db
+    df, feedback_col_names = lms.get_records_and_feedback(app_ids=[app_ids])
```

Comment:
this looks like we're passing in a list of list of strings, is that accurate?

Reply:
fixed -thanks